### PR TITLE
include formatting in lib

### PR DESCRIPTION
**Pull request by JoshuaSBrown**

### CODE REVIEW

**Comment by junghans** [comment]
The problem is that gromacs as well as eigen define `real`.

File: src/libcsg/beadmotifalgorithms.cc [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2009-2020 The VOTCA Development Team (http://www.votca.org)
+ * Copyright 2009-2019 The VOTCA Development Team (http://www.votca.org)
```
Comment (junghans) [inline src/libcsg/beadmotifalgorithms.cc]: Huh?

File: src/libcsg/cgmoleculedef.cc [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2009-2020 The VOTCA Development Team (http://www.votca.org)
+ * Copyright 2009-2019 The VOTCA Development Team (http://www.votca.org)
```
Comment (junghans) [inline src/libcsg/cgmoleculedef.cc]: Huh?

File: src/libcsg/beadmotifalgorithms.cc (hunk 1 above)
Comment (JoshuaSBrown) [reply src/libcsg/beadmotifalgorithms.cc]: That's weird. I might have a old version of the git pre-commit hook in that repo.

**Comment by JoshuaSBrown** [comment]
> The problem is that gromacs as well as eigen define `real`.

I figured that, that was the problem. I'm wondering how this should be handled, it is really annoying that the namespace scope is not being properly handled. What if we put all files from gromacs in their own namespace would this fix the problem.

```
namespace gromacs {
#include <gromacs...etc> 
}
```

I guess the real issue is that we are including a private header file to read in their .tpr files, which is dangerous and has lead to the problems we are currently experiencing. I could contact them and see if they would be willing to support a file reader in their actual api which is correctly scoped.

**Comment by junghans** [comment]
> > The problem is that gromacs as well as eigen define `real`.
> 
> I figured that, that was the problem. I'm wondering how this should be handled, it is really annoying that the namespace scope is not being properly handled. What if we put all files from gromacs in their own namespace would this fix the problem.
> 
> ```
> namespace gromacs {
> #include <gromacs...etc> 
> }
> ```
> 
> I guess the real issue is that we are including a private header file to read in their .tpr files, which is dangerous and has lead to the problems we are currently experiencing. I could contact them and see if they would be willing to support a file reader in their actual api which is correctly scoped.

Sure, go for it, maybe not as part of this PR.

**Review by junghans — CHANGES_REQUESTED** [review]
Add a changelog please.

**Review by junghans — APPROVED** [review]

**Pull request merged**